ballerina/response_formatter.bal

```diff
@@ -64,7 +64,7 @@ class ResponseFormatter {
                 } else if selection is parser:FieldNode {
                     __Type fieldType = self.getFieldType(selection.getName(), parentType, onType);
                     anydata|anydata[] fieldResult = ();
-                    if data.hasKey(selection.getAlias()) {
+                    if data.hasKey(selection.getAlias()) || data.hasKey(selection.getName()) {
```

Comment:
Isn't this lead to a bug? What if a certain field is required twice, once with an alias and once without an alias? Do we have a test for that?

Reply:
Do you mean a query like this? https://github.com/ballerina-platform/module-ballerina-graphql/pull/1440/files#diff-0219b63c80c6cac590137488c0e68e6491fe15e99f0305e41dd5643090116d17

Reply:
That's exactly what I was looking for!